src/data/learnQuizzes/questionBank.ts

```diff
@@ -0,0 +1,754 @@
+// Import data types
```

Comment:
Let's use kebab-case for directory names please (i.e. `data/learn-quizzes`). I'd even suggest we just go with `data/quizzes`).

Reply:
Agreed, can update